Tasks/FoobarAzureV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
All files under String folder are auto-generated and will be created when building the task.